package/MDAnalysis/lib/util.py

```diff
@@ -374,7 +375,12 @@ def _get_stream(filename, openfunction=open, mode='r'):
     """Return open stream if *filename* can be opened with *openfunction* or else ``None``."""
     try:
         stream = openfunction(filename, mode=mode)
-    except IOError:
+    except IOError as err:
+        t, v, tb = sys.exc_info()
+        if err.errno == 13:
+            six.raise_from( v, tb)
+        elif err.errno == 2:
+            six.raise_from(v, tb)
```

Comment:
What if `err.errno` is not 2 nor 13? Also, if there is no difference between the two branches, they can be merged.